### DISCUSSION TITLE
Git doppelte Ordner / Vercel Test schlägt fehl...

### DISCUSSION THREAD

**Discussion by Efendi5** [neuefische/questions#558] [Web] [answered]
Hallo,
gerade arbeite ich an meinem Capstone Projekt und kann aus folgendem Grund nicht mergen. Die Ordner sind alle doppelt einmal in upperCase und einmal in LowerCase.

Ich habe meinen aktuellen Stand committed und gepusht. [Dieser ist unter diesem Link erreichbar.][(https://github.com/mbosselmann/capstone-template-test/commit/d28b15bfea07ab0ce4839739aadaa35843ee89ec)](https://github.com/Efendi5/capstone-project/tree/AnswerOptions) Der Branch heißt AnswerOptions.

Beschreibung des Fehlers:
Durch den Fehler funktioniert der Test bei Git nicht und ich kann bei Vercel das Projekt nicht deployn. 
Folgende Befehle habe ich schon versucht:
git config core.ignorecase false
git rm -r --cached components/QuizCardPage/
git rm -r components/QuizCardPage/
git mv -f components/QuizCardPage/ components/quizCardPage

Vielen Dank für eure Hilfe!

**Comment by PReichetanz** [comment]
Hallo Efendi,

ich würde vorschlagen, die Dateien / Ordner, die Du nicht brauchst, einfach in VSCode per Rechtsklick zu löschen. Danach committest und pusht Du die Änderungen (aka dass Du Dateien gelöscht hast).

Dadurch hast Du doppelte Ordner / Dateien entfernt, die `git` (manchmal?) nicht voneinander unterscheiden kann.

**Reply by Efendi5** [reply]
Die Dateien gibt es in VSCode gar nicht.

**Reply by Gblur** [reply]
Problem hier ist, dass die Ordner im Gittree getracked sind, jedoch nicht in seinem Workspace vorhanden.
D.h. er kann sie nicht ohne weiteres loeschen sondern muss sie aus dem git tree entfernen

**Comment by djfarly** [comment, marked as answer]
We fixed it by using git mv to a temp directory and commiting, then doing it again…

**Reply by djfarly** [reply]
We essentially did this:

```
git mv components/QuizCardPage/ temp/
git add .
git commit -m "Move QuizCardPage to temporary directory"
git mv temp/ components/quizCardPage
git add .
git commit -m "Move QuizCardPage to lowercase directory"
```
